src/page-modules/assistant/details/index.tsx

```diff
@@ -12,8 +18,8 @@ export type AssistantDetailsProps = {
 
 export function AssistantDetails({ tripPattern }: AssistantDetailsProps) {
   const { t } = useTranslation();
-  const fromName = tripPattern.legs[0].fromPlace.name;
-  const toName = tripPattern.legs[tripPattern.legs.length - 1].toPlace.name;
+  const { fromName, toName, weekdayAndDate, timeRange, tripDuration } =
+    useTripDetailsHeader(tripPattern);
```

Comment:
Can tripPattern be null?

Reply:
The server can return null, but if so we will return the `EmptyMessage` component instead of `AssistantDetails`. 

https://github.com/AtB-AS/planner-web/blob/main/src/pages/assistant/%5Bid%5D.tsx#L19